requirements.txt

```diff
@@ -67,160 +44,74 @@ flake8-return==1.2.0 ; python_version >= "3.8" and python_version < "3.12"
 measurement==3.2.0 ; python_version >= "3.8" and python_version < "3.12"
-mistune==2.0.4 ; python_version >= "3.8" and python_version < "3.12"
-mock==4.0.3 ; python_version >= "3.8" and python_version < "3.12"
 mpmath==1.2.1 ; python_version >= "3.8" and python_version < "3.12"
 multidict==6.0.2 ; python_version >= "3.8" and python_version < "3.12"
-multiprocess==0.70.14 ; python_version >= "3.8" and python_version < "3.12"
 mypy-extensions==0.4.3 ; python_version >= "3.8" and python_version < "3.12"
-nbclient==0.5.13 ; python_version >= "3.8" and python_version < "3.12"
-nbconvert==7.2.4 ; python_version >= "3.8" and python_version < "3.12"
-nbdime==3.1.1 ; python_version >= "3.8" and python_version < "3.12"
-nbformat==5.7.0 ; python_version >= "3.8" and python_version < "3.12"
-nest-asyncio==1.5.6 ; python_version >= "3.8" and python_version < "3.12"
 numpy==1.23.4 ; python_version < "3.12" and python_version >= "3.8"
-outcome==1.2.0 ; python_version >= "3.8" and python_version < "3.12"
 packaging==21.3 ; python_version >= "3.8" and python_version < "3.12"
 pandas==1.5.1 ; python_version >= "3.8" and python_version < "3.12"
-pandocfilters==1.5.0 ; python_version >= "3.8" and python_version < "3.12"
-parso==0.8.3 ; python_version >= "3.8" and python_version < "3.12"
 pastel==0.2.1 ; python_version >= "3.8" and python_version < "3.12"
 pathspec==0.10.1 ; python_version >= "3.8" and python_version < "3.12"
 pbr==5.11.0 ; python_version >= "3.8" and python_version < "3.12"
-percy==2.0.2 ; python_version >= "3.8" and python_version < "3.12"
-pexpect==4.8.0 ; python_version >= "3.8" and python_version < "3.12" and sys_platform != "win32"
-pickleshare==0.7.5 ; python_version >= "3.8" and python_version < "3.12"
 pint==0.17 ; python_version >= "3.8" and python_version < "3.12"
 pip-licenses==3.5.5 ; python_version >= "3.8" and python_version < "3.12"
-pkgutil-resolve-name==1.3.10 ; python_version >= "3.8" and python_version < "3.9"
 platformdirs==2.5.3 ; python_version >= "3.8" and python_version < "3.12"
 plotly==5.11.0 ; python_version >= "3.8" and python_version < "3.12"
-pluggy==1.0.0 ; python_version >= "3.8" and python_version < "3.12"
 poethepoet==0.16.4 ; python_version >= "3.8" and python_version < "3.12"
-prometheus-client==0.15.0 ; python_version >= "3.8" and python_version < "3.12"
-prompt-toolkit==3.0.32 ; python_version >= "3.8" and python_version < "3.12"
-psutil==5.9.4 ; python_version >= "3.8" and python_version < "3.12"
 ptable==0.9.2 ; python_version >= "3.8" and python_version < "3.12"
-ptyprocess==0.7.0 ; python_version >= "3.8" and python_version < "3.12" and sys_platform != "win32" or python_version >= "3.8" and python_version < "3.12" and os_name != "nt"
-pure-eval==0.2.2 ; python_version >= "3.8" and python_version < "3.12"
-py==1.11.0 ; python_version >= "3.8" and python_version < "3.12"
-pybufrkit==0.2.19 ; python_version >= "3.8" and python_version < "3.12"
```

Comment:
Not sure if `pybufrkit` should stay?

Reply:
Currently, it is included into the new "test" dependency group, that's why it does not get included into the `requirements.txt` anymore. Let me know if you think it should, then we'll have to adjust some bits.

Reply:
I thin its fine if it's not included.